### DISCUSSION TITLE
Music streaming!

### DISCUSSION THREAD

**Discussion by spetterman66** [copy/v86#1042] [Show and tell]
# Screenshot
![image](https://github.com/copy/v86/assets/69303239/9a3892c1-5b81-4d1d-ba78-c3a5c0be3cd1)
# How to get started
1. Start Windows 98 or 2000 and enable networking
2. Download SoulSeek NS from slsknet.org, it works in internet explorer: http://www.slsknet.org/news/node/683
3. Install it
4. Locate it: Start => Programs => SoulSeek NS
5. Make up a random username and password
6. Download a song
7. Select a random folder if it asks you to share a folder
8. Wait until it's finished downloading
9. Open it by double-clicking it
10. You're finished! You have successfully acheived music streaming on v86.

**Comment by spetterman66** [comment]
(Thanks to @SuperMaxusa)
On v86, you can also use SoulSeek on the Windows 2000 profile. However, the steps are more complicated.
1. Boot up your machine
2. Double-click networking.bat
3. Open the control panel (Start => Run => "control")
4. Add/Remove devices
5. Add a device
6. Add a device not specified in list
7. Select **Sound Blaster 16 or AWE32 or compatible (WDM)** from the list of audio devices manufactured by Creative Technology Ltd.
8. Open Internet Explorer and navigate to http://slsknet.org
9. Disregard the JavaScript error
10. Scroll down until you see "Main Menu"
11. Click on "Download"
12. There should be an option to download SoulSeek NS at the bottom of the download page. Click it.
13. Download SoulSeek NS to the desktop, then double-click it when it's done
14. Finish the installation
15. Start => Programs => SoulSeek NS => SoulSeek
16. Open it, then make up a random username and password
17. Share the desktop folder when asked
18. Search for a song, sort by file size (ascending)
19. Double-click a mp3 or wav file, as the old version of Windows Media Player does not support FLAC files
20. Wait until it's finished downloading, then double-click the mp3 or wav file.

**Comment by Quantamyt** [comment]
I was forced to see this thread by @spetterman66

**Reply by spetterman66** [reply]
lmao k

**Comment by ProgrammerIn-wonderland** [comment]
Piracy on v86, we truly live in the future

**Reply by spetterman66** [reply]
~~There's no way actual streaming services would ever work on v86 lmao~~ Proven wrong by supermaxusa in the gitter on 20 jan '25
